### ISSUE TITLE
Prototype's getter setter function into class normal functions

### ISSUE THREAD

**Issue by RajaJaganathan** [lebab/lebab#275]
Prototype's getter setter function turns into normal functions in the class

### Input:

```
function Rectangle (height, width) {
    this.height = height;
    this.width = width;
}

Rectangle.prototype = {
  // Getter
  get area() {
    return this.calcArea();
  },
  // Method
  calcArea() {
    return this.height * this.width;
  }
}

const square = new Rectangle(10, 10);

console.log(square.area); // 100
```

### Output

```
class Rectangle {
  constructor(height, width) {
      this.height = height;
      this.width = width;
  }

  // Getter
  area() {
    return this.calcArea();
  }

  // Method
  calcArea() {
    return this.height * this.width;
  }
}

const square = new Rectangle(10, 10);

console.log(square.area); // f area() {return this.calcArea(); }
```

The expectation here is any getter/setter available in prototype object which also needs to preserve in ES6 Class too.

**Comment by nene** [comment]
Oh... This is definitely a bug. Thanks for reporting. Will investigate.

**Comment by nene** [comment]
Should be fixed now.

**Comment by nene** [comment]
Fix released in 3.0.1
